app/src/main/java/com/gh4a/utils/ApiKeyHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.gh4a.utils;
+
+import com.gh4a.BuildConfig;
+
+public class ApiKeyHelper {
+    private static final String GITHUB_FDROID_CLIENT_ID = "SOMETEXT";
+    private static final String GITHUB_FDROID_SECRET = "SOMETEXT";
```

Comment:
Is there no way to achieve this with a pure gradle solution, that is, put the hardcoded values in build.gradle and use it depending on the flavor there?

Also, publishing the client secret allows other people to impersonate Octodroid, which doesn't seem like a good idea to me. Isn't there a way to put the additional properties file into the F-Droid metadata? From a quick glance over other app's metadata, it looks like patches can be supplied, so it should be possible to submit a patch adding the properties file into their repo.